media/css/firefox/all.less

```diff
@@ -276,6 +276,7 @@ body.nightly {
     }
 
     #language-search {
+        width: @widthTablet - (@gridGutterWidth * 2 * 2.25);
```

Comment:
there is something wrong with the width of `#language-search`. `.search-column` sdoesn't seems to slove the problem I'm trying to solve.